src/test/com/cqfn/degitx/simulator/RequestPathTest.kt

```diff
@@ -1,21 +1,17 @@
 package com.cqfn.degitx.simulator
 
 import org.testng.Assert
-import org.testng.annotations.BeforeTest
 import org.testng.annotations.Test
 
 
 class RequestPathTest {
 
     private var graph = GraphGenerator()
 
-    @BeforeTest
-    fun setUp() {
-        graph.generate(GraphGenerator.Settings())
-    }
-
     @Test
     fun requestGoesToTarget() {
+        graph.generate(GraphGenerator.Settings())
+
```

Comment:
It could be moved to @BeforeSuite fun as in other test classes

Reply:
It was so, but does not work properly - test failed. I don't found out the reason for now, so this is a temporary solution.